js/forum/src/components/PostMeta.js

```diff
@@ -35,6 +35,7 @@ export default class PostMeta extends Component {
         <div className="Dropdown-menu dropdown-menu">
           <span className="PostMeta-number">{app.translator.trans('core.forum.post.number_tooltip', {number: post.number()})}</span>{' '}
           {fullTime(time)}
+          {post.data.attributes.canViewIp && post.data.attributes.userIp ? ' @ ' + post.data.attributes.userIp : ""}
```

Comment:
Sorry to nitpick, but I'd prefer single quotes for that empty string - to stay consistent... :)